<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceMaskDetector</class>
 <widget class="QDialog" name="FaceMaskDetector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>22</width>
     <height>431</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mVertical">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QOpenGLWidget" name="mVideo"/>
    </item>
    <item>
     <widget class="QPushButton" name="mOpen">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>521</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="tVertical">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QOpenGLWidget" name="tVideo"/>
    </item>
    <item>
     <widget class="QPushButton" name="tOpen">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
